cumulusci/tasks/metadata_etl/tests/test_help_text.py

```diff
@@ -0,0 +1,405 @@
+import pytest
+
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.tasks.salesforce.tests.util import create_task
+from cumulusci.tasks.metadata_etl.help_text import AddHelpText
+from cumulusci.utils.xml import metadata_tree
+
+# Custom Object with 2 custom fields with all elements present
+OBJECT_XML = b"""<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fields>
+        <fullName>Buster__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Indicates that the Contact is allowed to receive information protected by FERPA and other privacy laws, regulations, and policies.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>Indicates that the Contact is allowed to receive information protected by FERPA and other privacy laws, regulations, and policies.</inlineHelpText>
+        <label>FERPA Approved</label>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <fields>
+        <fullName>Bluth__c</fullName>
```

Comment:
Now I am a huge Arrested Development fan, but in the context of corporate OSS, I'm not sure we should theme our unit tests. Can we use neutral identifiers like `Foo__c`?

Reply:
I was also wondering if there were TM concerns.